.github/workflows/testing.yml

```diff
@@ -14,8 +25,6 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
```

Comment:
Things I don't know: Do we still need to setup the python version if we are using images with specific python versions?